src/test/regress/expected/multi_mx_node_metadata.out

```diff
@@ -641,19 +641,32 @@ SELECT verify_metadata('localhost', :worker_1_port),
 ---------------------------------------------------------------------
 -- Don't drop the reference table so it has shards on the nodes being disabled
 DROP TABLE dist_table_1, dist_table_2;
-SELECT pg_catalog.citus_disable_node_and_wait('localhost', :worker_2_port);
+SELECT pg_catalog.citus_disable_node('localhost', :worker_2_port);
 NOTICE:  Node localhost:xxxxx has active shard placements. Some queries may fail after this operation. Use SELECT citus_activate_node('localhost', 57638) to activate this node back.
- citus_disable_node_and_wait
+ citus_disable_node
 ---------------------------------------------------------------------
 
 (1 row)
 
-SELECT verify_metadata('localhost', :worker_1_port);
- verify_metadata
+SELECT wait_until_metadata_sync(30000);
+ wait_until_metadata_sync
 ---------------------------------------------------------------------
- t
+
 (1 row)
 
+-- show that node metadata is the same
+-- note that we cannot use verify_metadata here
+-- because there are several shards/placements
+-- in the metadata that are manually modified on the coordinator
+-- not on the worker, and pg_catalog.citus_disable_node does
+-- not sync the metadata
+SELECT result FROM run_command_on_workers($$SELECT jsonb_agg(row_to_json(row(pg_dist_node.*))) FROM pg_dist_node$$) WHERE nodeport=:worker_1_port
```

Comment:
Due to manual metadata changes the placement metadata does match on the workers. In the end, we are only interested in the node metadata after `citus_disable_node`, and hence only checking that.

I don't think it is worth to find which manual metadata change breaks `verify_metadata` at this point